CMSgov/mint-app
# EASI-2685 Analyzed Audit  to User Table Migrations

# [EASI-2685](https://jiraent.cms.gov/browse/EASI-2685)

## Changes and Description

- Migrated Analyzed Audit to use the user table
   - Note this previously was using a fake EUAID WRKR to denote a worker made the analyzed audit. I made a choice to reference the MINT System Account instead of making a separate user account for the faktory worker. If anyone thinks a separate account would be easier, we can easily implement that



## How to test this change
1. Make sure the unit tests pass
2. If you want to be more thorough, run the application with faktory jobs processing.
3. Verify you get the daily digest emails. 


## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
